packages/base/SkillCard/boxel-coding.json

```diff
@@ -2,7 +2,7 @@
..700;1,400..700&family=Playfair+Display:ital,wght@0,400..900;1,400..900&display=swap');\n        article {\n          --markdown-font-size: 1rem;\n          --markdown-font-family: var(--blog-post-font-family, 'Lora', serif);\n          --markdown-heading-font-family: var(\n            --blog-post-heading-font-family,\n            'Playfair Display',\n            serif\n          );\n          height: max-content;\n          min-height: 100%;\n          padding: var(--boxel-sp-sm) var(--boxel-sp-xl) var(--boxel-sp-lg);\n          background-color: #fcf9f2;\n          font-family: var(--blog-post-font-family, 'Lora', serif);\n        }\n        h1,\n        h2,\n        h3,\n        h4,\n        h5,\n        h6 {\n          font-family: var(\n            --blog-post-heading-font-family,\n            'Playfair Display',\n            serif\n          );\n        }\n        h1 {\n          font-size: 2.5rem;\n          font-weight: 600;\n          line-height: 1.25;\n          letter-spacing: normal;\n          margin-bottom: var(--boxel-sp-lg);\n          margin-top: var(--boxel-sp-xl);\n        }\n        .article-header {\n          margin-bottom: var(--boxel-sp-lg);\n        }\n        .featured-image :deep(.image) {\n          border-radius: var(--boxel-border-radius-xl);\n          overflow: hidden;\n        }\n        .blog {\n          background-color: inherit;\n        }\n        .blog + .featured-image {\n          margin-top: var(--boxel-sp-sm);\n        }\n        .description {\n          font-size: 1.25rem;\n          font-style: italic;\n        }\n        .info {\n          display: inline-flex;\n          align-items: center;\n          gap: var(--boxel-sp-xs);\n          flex-wrap: wrap;\n          list-style: none;\n          margin: 0;\n          padding: 0;\n        }\n        .info > li + li {\n          border-left: 1px solid black;\n          padding-left: var(--boxel-sp-xs);\n        }\n        .byline {\n          display: inline-flex;\n          align-items: center;\n          gap: var(--boxel-sp-xs) var(--boxel-sp);\n          flex-wrap: wrap;\n        }\n        .author {\n          display: contents; /* workaround for removing block-levelness of atom format */\n        }\n        .author-embedded-bio {\n          margin-top: var(--boxel-sp-xl);\n        }\n        .categories {\n          margin-top: var(--boxel-sp);\n          display: flex;\n          flex-wrap: wrap;\n          gap: var(--boxel-sp-xxs);\n        }\n        .featured-image + .categories {\n          margin-top: var(--boxel-sp-xl);\n        }\n        .category {\n          display: inline-block;\n          padding: 3px var(--boxel-sp-xxxs);\n          border-radius: var(--boxel-border-radius-sm);\n          font: 500 var(--boxel-font-xs);\n          letter-spacing: var(--boxel-lsp-sm);\n        }\n      </style>\n    </template>\n  };\n}\n```\n\n <style> tag must be at the root of the <template>, otherwise the app will break. It must not be nested.",
```

Comment:
Added: `<style> tag must be at the root of the <template>, otherwise the app will break. It must not be nested.`